configuration/concepts/drugs_concepts_void.csv

```diff
@@ -1,12 +1,4 @@
 Uuid,Void/Retire,Fully specified name:en,_version:1,_order:50
 ,TRUE,Tablet,,
 ,TRUE,Tablet(s),,
-,TRUE,Solution,,

```

Comment:
Same thing as above, let's make sure they are not created by Bahmni itself (**liquibase.xml**)

Reply:
Done